### DISCUSSION TITLE
I hope that the cascade model will be supported soon. I think not everyone is adaptable to comfui’s workflow. ​

### DISCUSSION THREAD

**Discussion by zlzzzlll** [lllyasviel/stable-diffusion-webui-forge#473] [General]
I hope that the cascade model will be supported soon. I think not everyone is adaptable to comfui’s workflow.
​

**Comment by E2GO** [comment]
Looks like it's in plan: https://github.com/lllyasviel/stable-diffusion-webui-forge/issues/234

**Comment by VeteranXT** [comment]
Minimal Requirements for cascade is 12GB VRAM.
